mscore/musescore.cpp

```diff
@@ -5864,13 +5865,23 @@ void MuseScore::cmd(QAction* a)
             return;
             }
       if (cmdn == "palette-search") {
+            // TODO: use new search box, or rename command
+            showPalette(true);
+            if (paletteWidget)
+                  paletteWidget->setFocus();
```

Comment:
@dmitrio95 I don't understand QML enough to know how to access the search box directly, but that's what we should really do here I assume.  Bonus points if pressing the command while focus is there again returns focus to the scoreview.